sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -257,7 +257,7 @@ AZ_NODISCARD az_result az_iot_pnp_client_sas_get_password(
  * @note This topic can also be used to set the MQTT Will message in the Connect message.
  *
  * @param[in] client The #az_iot_pnp_client to use for this call.
- * @param[in] component_name An #az_span specifying the component name to publish telemetry on.
+ * @param[in] component_name An #az_span specifying the component name to publish telemetry on. If
```

Comment:
????

Reply:
ahh odd didn't realize this was on there. I was going to address Johns comment from the previous PR here. Updating